### DISCUSSION TITLE
Inability to change machine type on codespaces

### DISCUSSION THREAD

**Discussion by baalajimaestro** [community/community#4280] [Codespaces] [answered]
Hi, I am trying to change my current machine type on Github Codespaces and its apparently not showing me options for Basic or Premium machine. On some cases, I would like to conserve the compute resources by migrating to Basic, since I am simply editing and pushing code up. I attempted creating a new codespace, as the warning on github said, but to no avail. Even the new codespace simply stuck to Standard size when I really didnt need it. And at times when I need the Premium I am not able to scale up either.

Link with screenshots of the issue: https://imgur.com/a/kQMjmk3

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @baalajimaestro Apologies for the delay. 
The current behavior is the following: 
- If you are not an org member and creating a Codespace off a public org repo, by default you only have access to the Standard SKU and hence you won't be able to modify the SKU to Premium. 
- If you are an org member, then you should be able to modify the size to Basic or Premium. 

Hope this helps.

**Comment by galen211** [comment]
Hi @tanmayeekamath - I have followed these instructions and am unable to select a machine type greater than the standard SKU.  I'm an enterprise subscriber.
